app/src/androidTest/java/com/github/se/travelpouch/ui/dashboard/TravelActivityScreen.kt

```diff
@@ -96,4 +97,40 @@ class TravelActivityScreen {
     composeTestRule.onNodeWithTag("activityItem").assertTextContains(activity.location.name)
     composeTestRule.onNodeWithTag("activityItem").assertTextContains("1/1/1970")
   }
+
+  @Test
+  fun verifyBannerIsDisplayedCorrectly() {
```

Comment:
Just for the test to be fully correct, I would add a last activity, with a occurrence date that is way beyond the 24 next hours, and verify that it is not displayed.

Reply:
Fair enough